pkg/util/http_utils_test.go

```diff
@@ -0,0 +1,74 @@
+package util
+
+import (
+	"github.com/jenkins-x/jx/src/github.com/stretchr/testify/assert"
```

Comment:
shouldn't this be "github.com/stretchr/testify/assert"?

Reply:
possibly, although my IDE provided multiple suggestions and it was not clear which one to choose.  I'll try the one you have suggested.

Reply:
yeah worth a try

Reply:
that did the trick, thanks!

Reply:
Otherwise you are using a transitive dependency rather than a direct dependency